src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -65,55 +66,65 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
     >
       <Grid container spacing={1}>
         <Grid item xs={4}>
-          <DetailDisplay label="First Name" content={recipient.firstName}></DetailDisplay>
+          <Box display="flex" alignItems="left">
+            <DetailDisplay label={'First Name'} content={recipient.firstName} copyButton={true} />
```

Comment:
you can just do 'copyButton' and it'll know you mean copyButton={true}, cool lil react thing, make sure to update every instance in this file